wp-content/themes/nuvanx-medical/inc/nvx-signature-phase-pages.php

```
@@ -78,85 +78,85 @@ function nvx_signature_phase_catalog(): array {
             'phase' => 2,
             'slug' => 'grasa-localizada-abdomen-flancos-madrid',
             'title' => 'Esa grasa del abdomen que no se va ni a dieta ni a gimnasio.',
-            'kicker' => 'NUVANX CONTOUR ARCHITECTURE™',
+            'kicker' => $upper,
```

PHPMD (2.15.0)

[warning] Avoid using undefined variables such as '$upper' which will lead to PHP notices. (undefined)

(UndefinedVariable)